tests/pages/page.py

```diff
@@ -29,6 +29,12 @@ def url(self):
     def wait_for_page_to_load(self):
         return self
 
+    def is_element_present(self, locator):
+        try:
+            return self.selenium.find_element(*locator)
+        except:
```

Comment:
We should catch the specific exception `NoSuchElementException` from `selenium.common.exceptions`.